lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
+      @_installDependencies
+      @_writeMychaConfig
+      @_writeTestHelper
+    ], done
+
+
+  _installDependencies: (done) =>
+    exec "npm install --save-dev chai sinon sinon-chai", {@cwd}, done
+
+
+  _writeMychaConfig: (done) =>
```

Comment:
In terms of gulp generating files, I've never seen it used for scaffolding. Do you have an example or know of what module to use? Its also for only two files so unless its simple to set up, I'd prefer just doing it by hand